src/components/BundleGroup/primitives.js

```diff
@@ -0,0 +1,18 @@
+import Flex from '../Flex';
+import Box from '../Box';
+import styled from '@emotion/styled';
+import { themeGet } from 'styled-system';
+import { StyledNakedButton } from '../Bundle/primitives';
+
+export const StyledBundleGroup = styled(Flex)`
+  gap: ${themeGet('space.5')};
+  flex-direction: column;
+  @media (min-width: ${themeGet('breakpoints.1')}) {
+    flex-direction: row;
+    ${StyledNakedButton} {
+      ${Box}:first-child {
+        padding-top: ${(props) => (props.applyPadding ? '44px' : '16px')};
```

Comment:
```suggestion
        padding-top: ${(props) => (props.variant === '?' ? themeGet('space.11') : themeGet('space.4'))};
```
Use theme for spacing instead?

Also, should the prop be renamed as it still applies padding when it is false? `variant` is just an example, there could be better name to describe how it changes the component

Reply:
no worries - updated in next commit 🙏